src/Artsy/Router/Boot.tsx

```diff
@@ -57,32 +57,30 @@ export class Boot extends React.Component<BootProps> {
     }
 
     return (
-      <ErrorBoundary>
-        <HeadProvider headTags={headTags}>
-          <StateProvider>
-            <Artsy.SystemContextProvider {...contextProps}>
-              <MediaContextProvider onlyMatch={props.onlyMatchMediaQueries}>
-                <ResponsiveProvider
-                  mediaQueries={themeProps.mediaQueries}
-                  initialMatchingMediaQueries={
-                    props.onlyMatchMediaQueries as any
-                  }
-                >
-                  <Theme>
-                    <Grid fluid>
+      <HeadProvider headTags={headTags}>
+        <StateProvider>
+          <Artsy.SystemContextProvider {...contextProps}>
+            <MediaContextProvider onlyMatch={props.onlyMatchMediaQueries}>
+              <ResponsiveProvider
+                mediaQueries={themeProps.mediaQueries}
+                initialMatchingMediaQueries={props.onlyMatchMediaQueries as any}
+              >
+                <Theme>
+                  <Grid fluid>
+                    <ErrorBoundary>
```

Comment:
Moved the `ErrorBoundary` down so that it starts at our component tree and can inherit styles